contracts/multisig-prover/src/msg.rs

```diff
@@ -42,6 +42,11 @@ pub enum QueryMsg {
     GetWorkerSet,
 }
 
+#[cw_serde]
+pub enum MigrateMsg {
+    UpdateConfig { new_config: InstantiateMsg },
+}
```

Comment:
I don't think enum is a good fit, because there is never going to be more than one allowed migration msg at a time. The way I see it, enums should only be used if there are multiple equally valid variant or it's clear the type needs to be expanded in the future.

Reply:
That's generally true, but if we just allow arbitrary updates to the config via migration, then that is always allowed, even if other migrations are allowed too. Updating the config can happen any number of times.